DevDependencies out-of-date

https://david-dm.org/dwyl/hapi-auth-jwt2?type=dev
![image](https://user-images.githubusercontent.com/194400/66720139-eca9dd80-edf0-11e9-948b-06e353adee0a.png)

# Todo

+ [ ] update `eslint` and any others that are out-of-date.

The main devDependency that is out-of-date is `eslint` 
but I'm going to attempt to update _all_ of them and see if CI passes. 👍

Build failed with latest version of `eslint`:
https://travis-ci.org/dwyl/hapi-auth-jwt2/jobs/597334114
![image](https://user-images.githubusercontent.com/194400/66720267-7a39fd00-edf2-11e9-9907-859271de8882.png)

```
/home/travis/build/dwyl/hapi-auth-jwt2/lib/index.js
  66:16  error  Parsing error: Unexpected token ..
```

# 😕

--- a/.eslintrc
+++ b/.eslintrc
@@ -121,9 +121,6 @@
         "node": true
     },
     "parserOptions": {
-      "ecmaVersion": 2017,
-      "ecmaFeatures": {
-          "experimentalObjectRestSpread": true
-      }
+      "ecmaVersion": 2018
     }
 }

--- a/package.json
+++ b/package.json
@@ -61,7 +61,7 @@
     "@hapi/basic": "^5.1.1",
     "@types/hapi__hapi": "^18.2.0",
     "aguid": "^2.0.0",
-    "eslint": "^5.9.0",
+    "eslint": "^6.6.0",
     "eslint-plugin-prettier": "^3.0.0",
     "nyc": "^14.1.1",
     "pre-commit": "^1.2.2",
@@ -70,7 +70,7 @@
     "tape": "^4.9.1"
   },
   "engines": {
-    "node": ">=8.9.0"
+    "node": ">=8.10.0"
   },
   "nyc": {
     "check-coverage": true,

